static/src/javascripts/projects/common/modules/ui/notification-counter.spec.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import mediator from 'lib/mediator';
+import { initNotificationCounter, _ } from './notification-counter';
+
+jest.mock('lib/mediator');
+
+describe('notification-counter', () => {
+    const ORIGINAL_PAGETITLE = document.title;
+
+    beforeEach(() => {
+        document.title = ORIGINAL_PAGETITLE;
+        initNotificationCounter();
+    });
+
+    it('should put a counter in the title bar', () => {
+        _.setCount(2);
```

Comment:
I think we can trigger this function by calling `mediator.emit('modules:autoupdate:unread', 2)`. Then we don't need to expose the internal `setCount` function